spec/pardot/objects/prospects_spec.rb

```diff
@@ -55,4 +55,47 @@ def sample_results
       assert_authorization_header
     end
   end
+
+  context 'for API version 4:' do
+    let(:sample_response) do
+      <<~RESPONSE
```

Comment:
any idea what an error response looks like? esp thinking about the weird partial failure state we dealt with on hubspot

Reply:
Definitely need to get some eyes on real requests here, because I have no idea what any of these responses look like yet 😅